doc/bql.rst

```diff
@@ -117,6 +117,74 @@ executing SQL instead of BQL in Bayeslite.
 
    FUTURE: Renaming columns (Github issue #35).
 
+Model Definition Language
+^^^^^^^^^^^^^^^^^^^^^^^^^
++++++++++++
+Populations
++++++++++++
+A population specifies which columns in a table should be modeled or ignored.
+For those that are modeled, it specifies whether they should be modeled as
+numerical or nominal.
+
+``GUESS SCHEMA FOR <name>``
+   Guess a population schema for the table *name*. A schema maps the columns in
+   *name* to heuristically guessed statistical types. Statistical types can be
+   guessed to be NOMINAL or NUMERICAL. Columns can also be guessed to be ignored
+   (IGNORE).
+
+``CREATE POPULATION [IF NOT EXISTS] FOR <table> WITH SCHEMA { [GUESS STATTYPES
+FOR (<column(s)>)] [MODEL <column(s)> AS <stattype>] [IGNORE <column(s)>] }``
+   Create a population for *table* with a schema defined by guessing the
+   statistical types for some or all columns (which can be referred to by \*)
+   and/or explicitly modeling columns using a particular statistical type
+   (NUMERICAL or NOMINAL) or ignoring them. If more than one schema definition
+   is used, they should be separated by semicolons.
+
+``DROP POPULATION [IF EXISTS] <population>``
+   Drop the population *population* and all its contents.
+   Will fail if there are still metamodels associated with this population.
+
+``ALTER POPULATION <population> ADD VARIABLE <variable> [<stattype>]``
+   Add the variable *variable* to the population *population*. Specify that it
+   should be modeled with the statistical type *stattype* (optional), otherwise
```

Comment:
'Specify that it should be modeled with the statistical type *stattype* (optional); otherwise...'